Add Manipulation Functions to Paddle Backend

chinonsoonodi opened the pull request:
Issue #13873

MahmoudAshraf97 commented:
Hi, can you explain exactly what are you doing in this PR? the function is already implemented

chinonsoonodi commented:
> Hi, can you explain exactly what are you doing in this PR? the function is already implemented

Didn't see it when I chose it. Then I modified and removed the lambda

MahmoudAshraf97 commented:
Please Open a new PR for scatter flat, and make sure the issue is not assigned to someone else
